test/partisan_SUITE.erl

```diff
@@ -182,39 +200,53 @@ default_manager_test(Config) ->
                              Connections
                      end,
 
-    VerifyConnectionsFun = fun(Node, Parallelism) ->
+    VerifyConnectionsFun = fun(Node, Channel, Parallelism) ->
                                 %% Get list of connections.
                                 {ok, Connections} = ConnectionsFun(Node),
 
                                 %% Verify we have enough connections.
                                 dict:fold(fun(_N, Active, Acc) ->
-                                                 case length(Active) == Parallelism of
-                                                     true ->
-                                                         Acc andalso true;
-                                                     false ->
-                                                         Acc andalso false
-                                                 end
-                                         end, true, Connections)
+                                    Filtered = lists:filter(fun({_, C, _}) -> 
```

Comment:
More simple list comprehensions. Since this is exactly the same as the first one I commented on, you could extract this to a function, like "filter_to_channel" or something.